clang/lib/Parse/ParseDecl.cpp

```diff
@@ -220,8 +230,28 @@ void Parser::ParseGNUAttributes(ParsedAttributes &Attrs,
         continue;
       }
 
+      bool LateParse = false;
+      if (!LateAttrs)
+        LateParse = false;
+      else {
```

Comment:
Can remove a level of indentation here by making this an `else if` followed by an `else`.